modules/renter/contractor/contractor.go

```diff
@@ -140,7 +140,9 @@ func (c *Contractor) Contracts() []modules.RenterContract {
 
 // ContractUtility returns the utility fields for the given contract.
 func (c *Contractor) ContractUtility(id types.FileContractID) (modules.ContractUtility, bool) {
-	return c.staticContractUtility(id)
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	return c.contractUtility(id)
```

Comment:
technically should be a writelock, unless we call it `readlockContractUtility`.

I wish these locking conventions were just builtins with compiler enforcement